containers/Makefile

```diff
@@ -36,15 +36,8 @@ VERIFIER_SOURCE :=                           \
     $(VERIFIER_SOURCE_FROM_CONCENT_API)
 
 
-ifndef CONCENT_DIR
-    CONCENT_DIR:= $(abspath ../../concent)
-endif
-$(info CONCENT_DIR=$(CONCENT_DIR))
-
-ifndef GOLEM_DIR
-    GOLEM_DIR:= $(abspath ../../golem)
-endif
-$(info GOLEM_DIR=$(GOLEM_DIR))
+CONCENT_DIR:= build/repositories/concent/
+GOLEM_DIR:= build/repositories/golem/
```

Comment:
Please remove these variables. They can no longer be changed by the user who runs make. Leaving them here may imply that they can be changed.